pkg/test/kube/accessor.go

```diff
@@ -421,11 +424,24 @@ func (a *Accessor) CreateNamespaceWithInjectionEnabled(ns string, istioTestingAn
 	return err
 }
 
+func (a *Accessor) CreateNamespaceWithLabels(ns string, istioTestingAnnotation string, labels map[string]string) error {
```

Comment:
Could you add a brief comment for this method?

Reply:
done